### PR TITLE
🎁 Add `place-items` property

### PR DESCRIPTION
**Pull request by sergiofontes**
From [CSS-Tricks](https://css-tricks.com/almanac/properties/p/place-items/):
> The `place-items` property in CSS is shorthand for the `align-items` and `justify-items` properties, combining them into a single declaration.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -72,6 +72,10 @@ module.exports = {
                     'justify-self',
                 ],
             },
+            {
+                // Align + Justify.
+                properties: ['place-items'],
+            },
```
Comment (stormwarning) [inline index.js:78]: In this config the shorthand properties are ordered before the more specific ones. Could you combine the "align" and "justify" groups into a single one called "Layout alignment." that has `place-items` first, followed by the `align-` properties, and then the `justify-` ones?

**Review by stormwarning — CHANGES_REQUESTED** [review]
Thanks for the PR — I've been meaning to add this!

File: index.js (hunk 1 above)
Comment (sergiofontes) [reply index.js:78]: Sure thing.

**Review by stormwarning — APPROVED** [review]
Nice, thanks!

**Pull request merged**